INFO 2025-03-28 01:23:16,173 autoreload Watching for file changes with StatReloader
INFO 2025-03-28 01:23:45,201 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\myweb\settings.py changed, reloading.
INFO 2025-03-28 01:23:53,164 autoreload Watching for file changes with StatReloader
INFO 2025-03-28 01:49:49,671 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-28 01:49:50,107 autoreload Watching for file changes with StatReloader
INFO 2025-03-28 16:28:00,301 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-28 16:28:00,943 autoreload Watching for file changes with StatReloader
INFO 2025-03-28 16:50:12,791 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-28 16:50:13,244 autoreload Watching for file changes with StatReloader
INFO 2025-03-28 17:45:55,066 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-28 17:45:55,543 autoreload Watching for file changes with StatReloader
INFO 2025-03-28 22:16:52,000 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-28 22:16:52,627 autoreload Watching for file changes with StatReloader
INFO 2025-03-28 22:26:04,134 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\models.py changed, reloading.
INFO 2025-03-28 22:26:04,624 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:02:24,006 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-29 01:02:24,490 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:02:31,114 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-29 01:02:31,557 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:02:42,253 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-29 01:02:42,695 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:27:58,393 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-29 01:27:58,836 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:33:25,649 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-29 01:33:26,091 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:39:29,596 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-29 01:39:30,021 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:39:44,793 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-29 01:39:45,209 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:41:48,049 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\urls.py changed, reloading.
INFO 2025-03-29 01:41:48,486 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:42:10,580 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py changed, reloading.
INFO 2025-03-29 01:42:11,015 autoreload Watching for file changes with StatReloader
ERROR 2025-03-29 01:42:13,401 log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: store_product.motor_control_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\views.py", line 46, in store
    logger.info(f"Store page rendered with {len(products)} products")
                                            ~~~^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: store_product.motor_control_id
ERROR 2025-03-29 01:42:13,412 basehttp "GET / HTTP/1.1" 500 130503
INFO 2025-03-29 01:50:53,058 autoreload D:\Desktop\PROJECTEND ระบบชำระเงินสมบูรแล้วเผื่อในกรณีโค้ดผิดพลาด4\myweb\store\models.py changed, reloading.
INFO 2025-03-29 01:50:53,539 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:53:12,935 autoreload Watching for file changes with StatReloader
INFO 2025-03-29 01:53:14,315 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-03-29 01:53:14,362 basehttp "GET /static/css/main.css HTTP/1.1" 304 0
INFO 2025-03-29 01:53:14,367 basehttp "GET /static/images/cart.png HTTP/1.1" 304 0
INFO 2025-03-29 01:53:14,367 basehttp "GET /images/shoes_yNyuZHS.jpg HTTP/1.1" 304 0
INFO 2025-03-29 01:53:14,390 basehttp "GET /images/poster_TUOkDrn.jpg HTTP/1.1" 304 0
INFO 2025-03-29 01:53:14,391 basehttp "GET /images/headphones_YycFlsb.jpg HTTP/1.1" 304 0
INFO 2025-03-29 01:53:14,394 basehttp "GET /static/js/cart.js HTTP/1.1" 304 0
INFO 2025-03-29 01:53:16,069 basehttp "GET /cart/ HTTP/1.1" 200 6116
INFO 2025-03-29 01:53:16,099 basehttp "GET /static/images/arrow-up.png HTTP/1.1" 304 0
INFO 2025-03-29 01:53:16,099 basehttp "GET /static/images/arrow-down.png HTTP/1.1" 304 0
INFO 2025-03-29 01:53:16,900 basehttp "GET /checkout/ HTTP/1.1" 200 9977
INFO 2025-03-29 01:53:21,181 basehttp "POST /process_order/ HTTP/1.1" 200 275
INFO 2025-03-29 01:55:54,230 autoreload Watching for file changes with StatReloader
INFO 2025-03-31 23:24:52,026 autoreload Watching for file changes with StatReloader
INFO 2025-03-31 23:24:53,487 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-03-31 23:24:54,706 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-03-31 23:24:55,442 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-03-31 23:24:59,284 basehttp "POST /process_order/ HTTP/1.1" 200 275
INFO 2025-03-31 23:27:40,841 basehttp "POST /process_order/ HTTP/1.1" 200 275
INFO 2025-03-31 23:33:46,092 autoreload Watching for file changes with StatReloader
INFO 2025-03-31 23:34:36,794 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-03-31 23:34:37,390 autoreload Watching for file changes with StatReloader
INFO 2025-03-31 23:44:21,574 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-03-31 23:53:57,092 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-03-31 23:59:00,065 autoreload Watching for file changes with StatReloader
INFO 2025-03-31 23:59:02,797 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-03-31 23:59:03,868 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-03-31 23:59:04,946 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-03-31 23:59:09,265 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 00:23:17,236 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 00:23:17,813 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 00:29:48,542 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\urls.py changed, reloading.
INFO 2025-04-01 00:29:49,250 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 00:34:27,784 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\urls.py changed, reloading.
INFO 2025-04-01 00:34:28,411 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 00:34:54,520 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 00:34:55,046 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 00:41:39,728 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 00:41:41,539 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 00:41:42,426 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 00:41:46,392 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 00:43:06,251 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 00:43:13,113 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 00:44:09,371 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 00:44:13,899 basehttp "POST /process_order/ HTTP/1.1" 200 276
WARNING 2025-04-01 00:51:10,581 log Not Found: /webhook/
WARNING 2025-04-01 00:51:10,582 basehttp "POST /webhook/ HTTP/1.1" 404 4809
INFO 2025-04-01 00:53:59,547 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\urls.py changed, reloading.
INFO 2025-04-01 00:54:00,133 autoreload Watching for file changes with StatReloader
WARNING 2025-04-01 00:54:31,293 log Bad Request: /webhook/
WARNING 2025-04-01 00:54:31,293 basehttp "POST /webhook/ HTTP/1.1" 400 38
ERROR 2025-04-01 00:54:47,617 log Internal Server Error: /webhook/opn
Traceback (most recent call last):
  File "D:\Desktop\ProjectEnd2\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Desktop\ProjectEnd2\venv\Lib\site-packages\django\utils\deprecation.py", line 131, in __call__
    response = self.process_response(request, response)
  File "D:\Desktop\ProjectEnd2\venv\Lib\site-packages\django\middleware\common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Desktop\ProjectEnd2\venv\Lib\site-packages\django\middleware\common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
    ...<9 lines>...
    )
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 127.0.0.1:8000/webhook/opn/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
ERROR 2025-04-01 00:54:47,620 basehttp "POST /webhook/opn HTTP/1.1" 500 70891
WARNING 2025-04-01 00:54:52,334 log Bad Request: /webhook/
WARNING 2025-04-01 00:54:52,335 basehttp "POST /webhook/ HTTP/1.1" 400 38
INFO 2025-04-01 01:00:53,631 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 01:00:54,177 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 01:01:13,046 autoreload Watching for file changes with StatReloader
ERROR 2025-04-01 01:01:19,680 log Internal Server Error: /webhook/
ERROR 2025-04-01 01:01:19,681 basehttp "POST /webhook/ HTTP/1.1" 500 38
INFO 2025-04-01 01:02:29,401 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 01:39:55,269 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 01:40:00,786 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 01:40:01,963 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 01:40:04,530 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 01:40:08,070 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 01:41:05,634 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 01:41:06,832 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 01:41:08,129 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 01:41:09,686 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 01:41:10,121 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 01:41:13,681 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 01:50:59,863 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 01:51:00,446 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 01:51:04,170 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 01:51:13,769 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 01:51:15,039 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 01:51:16,099 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 01:51:16,691 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 01:51:20,569 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 01:55:14,792 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 01:55:15,390 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 01:55:18,058 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 01:55:18,995 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 01:55:19,611 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 01:55:25,242 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 02:13:44,841 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 02:13:45,426 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 02:22:22,755 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 02:22:23,327 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 02:27:29,295 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 02:27:29,932 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 02:34:51,022 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 02:34:51,567 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 02:37:12,289 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 02:37:12,856 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 02:49:50,366 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 02:49:51,007 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 02:51:59,003 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 02:52:00,644 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 02:52:02,861 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 02:52:03,502 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 02:52:07,524 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 02:59:29,846 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 03:01:26,671 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 03:01:28,532 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 03:01:29,420 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 03:01:34,474 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 03:33:19,439 autoreload Watching for file changes with StatReloader
WARNING 2025-04-01 03:33:21,042 log Bad Request: /webhook/
WARNING 2025-04-01 03:33:21,043 basehttp "POST /webhook/ HTTP/1.1" 400 32
INFO 2025-04-01 03:47:09,673 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 03:47:10,262 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 14:43:00,369 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 14:43:47,061 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\myweb\settings.py changed, reloading.
INFO 2025-04-01 14:43:47,631 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 14:44:38,226 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\myweb\settings.py changed, reloading.
INFO 2025-04-01 14:44:38,740 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 14:46:36,609 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 14:46:37,166 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 14:46:38,754 basehttp "GET / HTTP/1.1" 200 5173
WARNING 2025-04-01 14:46:38,864 log Not Found: /favicon.ico
WARNING 2025-04-01 14:46:38,865 basehttp "GET /favicon.ico HTTP/1.1" 404 5074
INFO 2025-04-01 14:46:40,011 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 14:46:40,891 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 14:46:45,265 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 15:04:51,390 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 15:06:21,745 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 15:06:22,252 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 15:06:23,914 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 15:06:26,063 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 15:06:26,876 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-01 15:06:31,038 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-01 15:51:50,988 basehttp "GET /payment_success/114/ HTTP/1.1" 200 3698
INFO 2025-04-01 15:55:03,059 basehttp "GET /payment_success/114/ HTTP/1.1" 200 3698
INFO 2025-04-01 16:06:32,258 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 16:06:32,851 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 16:06:39,470 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 16:06:56,368 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 17:18:35,267 autoreload D:\Desktop\โปรเจค\PROJECTEND ระบบชำระเงินสมบูรแล้วติดปัญหาคุมมอเตอร์ - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-01 17:18:35,878 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 17:18:37,531 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-01 17:18:39,087 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-01 20:33:51,146 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 20:33:59,027 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-01 20:33:59,484 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 21:43:44,444 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-01 21:43:44,916 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 22:45:19,471 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-01 22:45:19,948 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 23:39:04,841 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-01 23:39:05,315 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 23:39:32,988 autoreload Watching for file changes with StatReloader
INFO 2025-04-01 23:41:26,030 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-01 23:41:26,496 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 01:06:16,077 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 01:06:16,593 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 11:34:39,934 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 11:34:40,577 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 11:54:54,190 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 11:54:54,879 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 11:56:18,379 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 11:56:18,840 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 12:03:21,512 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 12:03:21,975 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 12:07:16,069 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-02 12:07:17,031 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-02 12:07:17,722 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-02 12:07:22,293 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-02 12:10:14,738 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 12:10:15,215 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 12:10:21,994 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 12:10:22,456 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 12:17:04,737 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 12:17:05,224 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 12:17:19,143 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 12:17:19,599 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 12:17:23,032 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 12:24:34,879 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 12:24:35,366 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 13:23:45,860 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 13:23:46,353 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 13:58:24,500 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 13:58:24,959 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 13:58:32,012 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-02 13:58:33,214 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-02 13:58:34,158 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-02 13:58:38,523 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-02 14:22:54,875 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 14:22:55,355 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 14:34:02,262 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 14:34:02,761 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 19:57:43,656 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\myweb\settings.py changed, reloading.
INFO 2025-04-02 19:58:07,500 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 19:59:30,905 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 19:59:31,360 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 20:07:46,635 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 20:07:47,091 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 20:08:49,624 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-02 20:09:08,623 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-02 20:09:08,814 basehttp "GET /static/css/main.css HTTP/1.1" 200 3856
INFO 2025-04-02 20:09:08,820 basehttp "GET /static/images/cart.png HTTP/1.1" 200 92001
INFO 2025-04-02 20:09:08,824 basehttp "GET /static/images/arrow-up.png HTTP/1.1" 200 18590
INFO 2025-04-02 20:09:08,828 basehttp "GET /static/images/arrow-down.png HTTP/1.1" 200 25650
INFO 2025-04-02 20:09:08,834 basehttp "GET /images/poster_TUOkDrn.jpg HTTP/1.1" 200 22934
INFO 2025-04-02 20:09:08,964 basehttp "GET /static/js/cart.js HTTP/1.1" 200 1757
INFO 2025-04-02 20:09:10,761 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-02 20:09:10,818 basehttp "GET /static/css/main.css HTTP/1.1" 200 3856
INFO 2025-04-02 20:09:10,846 basehttp "GET /images/poster_TUOkDrn.jpg HTTP/1.1" 200 22934
INFO 2025-04-02 20:09:10,847 basehttp "GET /static/images/cart.png HTTP/1.1" 200 92001
INFO 2025-04-02 20:09:10,874 basehttp "GET /static/js/cart.js HTTP/1.1" 200 1757
INFO 2025-04-02 20:09:16,073 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-02 20:09:43,431 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 20:09:54,785 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-02 21:37:31,100 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 21:37:31,608 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 21:37:40,441 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 22:31:38,802 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 22:31:39,227 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 22:54:40,491 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\utils.py changed, reloading.
INFO 2025-04-02 22:54:40,961 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 22:55:10,178 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\utils.py changed, reloading.
INFO 2025-04-02 22:55:10,613 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 23:03:54,750 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\myweb\settings.py changed, reloading.
INFO 2025-04-02 23:03:55,215 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 23:04:22,387 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\utils.py changed, reloading.
INFO 2025-04-02 23:04:22,821 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 23:04:24,464 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-02 23:04:25,436 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-02 23:04:26,324 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-02 23:04:30,431 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-02 23:15:20,446 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 23:15:25,124 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-02 23:31:57,759 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-02 23:31:58,224 autoreload Watching for file changes with StatReloader
INFO 2025-04-02 23:32:00,198 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-02 23:32:03,956 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-02 23:32:05,497 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-02 23:32:06,743 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-02 23:32:07,307 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-02 23:32:11,751 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-02 23:40:34,446 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\urls.py changed, reloading.
INFO 2025-04-02 23:40:34,905 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 02:00:59,922 autoreload Watching for file changes with StatReloader
ERROR 2025-04-03 02:01:38,401 log Internal Server Error: /webhook/opn/
ERROR 2025-04-03 02:01:38,402 basehttp "POST /webhook/opn/ HTTP/1.1" 500 38
ERROR 2025-04-03 02:02:04,243 log Internal Server Error: /webhook/opn/
ERROR 2025-04-03 02:02:04,243 basehttp "POST /webhook/opn/ HTTP/1.1" 500 38
INFO 2025-04-03 03:39:08,063 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-03 03:39:08,549 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 03:52:10,931 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-03 03:52:11,388 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 03:52:40,665 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-03 03:52:41,136 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 17:01:15,006 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 17:45:40,037 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-03 17:45:40,533 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 19:52:21,429 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-03 19:52:21,963 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 19:52:29,971 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 20:38:06,069 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\myweb\settings.py changed, reloading.
INFO 2025-04-03 20:38:06,555 autoreload Watching for file changes with StatReloader
INFO 2025-04-03 22:05:57,915 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500\myweb\store\views.py changed, reloading.
INFO 2025-04-03 22:05:58,476 autoreload Watching for file changes with StatReloader
INFO 2025-04-04 20:52:05,704 autoreload Watching for file changes with StatReloader
INFO 2025-04-04 20:52:07,073 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-04 20:52:08,038 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-04 20:52:08,786 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-04 20:52:12,573 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-04 23:57:55,570 autoreload Watching for file changes with StatReloader
INFO 2025-04-04 23:57:56,953 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-04 23:57:57,871 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-04 23:57:58,537 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-04 23:58:02,562 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-05 00:07:08,979 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 00:07:10,329 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-05 00:07:11,347 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-05 00:07:11,946 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-05 00:07:15,784 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-05 00:12:39,872 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 00:12:40,339 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 00:13:47,432 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 00:13:47,898 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 00:14:46,819 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 00:14:47,301 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 00:15:04,360 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-05 00:15:05,206 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-05 00:15:07,170 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-05 00:15:11,388 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-05 00:16:47,883 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 00:16:50,785 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-05 00:16:58,163 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-05 00:51:05,047 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 00:51:06,375 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-05 00:51:07,406 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-05 00:51:08,334 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-05 00:51:12,105 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-05 01:24:31,584 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 01:24:32,104 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 01:24:42,432 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 01:24:42,848 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 01:24:47,610 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 01:24:48,070 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 01:26:02,503 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 01:26:02,935 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 01:26:05,740 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-05 01:26:06,788 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-05 01:26:07,442 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-05 01:26:13,833 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-05 01:35:43,493 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 01:35:43,982 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 01:48:03,994 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 01:48:04,462 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:13:51,061 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:13:51,515 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:16:50,017 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:16:50,487 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:19:05,133 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:19:05,606 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:21:33,479 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:21:33,953 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:25:51,063 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:25:51,567 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:35:58,203 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:35:58,677 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:38:39,782 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:38:40,256 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:42:47,216 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:42:47,686 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 02:47:29,402 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 02:47:29,869 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 03:07:36,317 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 03:07:36,781 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 03:49:37,932 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 03:49:38,431 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 03:49:43,206 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 03:49:43,665 autoreload Watching for file changes with StatReloader
INFO 2025-04-05 04:09:59,469 autoreload D:\Desktop\PROJECTEND ติดปัญหาแค่ 500 ไม่มีเวอชั้น\myweb\store\views.py changed, reloading.
INFO 2025-04-05 04:09:59,962 autoreload Watching for file changes with StatReloader
INFO 2025-04-06 20:20:10,270 autoreload Watching for file changes with StatReloader
INFO 2025-04-06 20:20:13,198 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-06 20:20:14,238 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-06 20:20:15,627 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-06 20:20:16,406 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-06 20:20:20,571 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-21 16:40:00,158 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 16:40:01,967 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-21 16:40:03,889 basehttp "GET /cart/ HTTP/1.1" 200 6117
INFO 2025-04-21 16:40:04,576 basehttp "GET /checkout/ HTTP/1.1" 200 9977
INFO 2025-04-21 16:40:08,360 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-21 17:02:07,081 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 17:02:11,794 basehttp "GET / HTTP/1.1" 200 5170
INFO 2025-04-21 17:02:13,456 basehttp "GET /cart/ HTTP/1.1" 200 4254
INFO 2025-04-21 17:02:14,174 basehttp "GET /checkout/ HTTP/1.1" 200 8980
INFO 2025-04-21 17:03:47,143 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\models.py changed, reloading.
INFO 2025-04-21 17:03:47,678 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 17:18:33,093 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 17:18:34,485 basehttp "GET / HTTP/1.1" 200 5173
INFO 2025-04-21 17:18:38,602 basehttp "GET /cart/ HTTP/1.1" 200 6117
INFO 2025-04-21 17:18:39,527 basehttp "GET /checkout/ HTTP/1.1" 200 9977
INFO 2025-04-21 17:18:44,450 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-21 17:59:56,726 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 17:59:58,145 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 17:59:59,806 basehttp "GET /cart/ HTTP/1.1" 200 6118
INFO 2025-04-21 18:00:00,487 basehttp "GET /checkout/ HTTP/1.1" 200 9978
ERROR 2025-04-21 18:00:08,580 log Internal Server Error: /process_order/
ERROR 2025-04-21 18:00:08,581 basehttp "POST /process_order/ HTTP/1.1" 500 50
INFO 2025-04-21 18:00:27,077 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-21 18:00:27,540 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 18:00:29,501 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 18:00:31,897 basehttp "GET /cart/ HTTP/1.1" 200 6118
INFO 2025-04-21 18:00:33,806 basehttp "GET /checkout/ HTTP/1.1" 200 9978
INFO 2025-04-21 18:00:39,693 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-21 18:08:39,823 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 18:08:44,461 basehttp "GET /cart/ HTTP/1.1" 200 6118
INFO 2025-04-21 18:08:45,988 basehttp "GET /checkout/ HTTP/1.1" 200 9978
INFO 2025-04-21 18:11:20,072 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 18:11:22,130 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 18:11:23,927 basehttp "GET /cart/ HTTP/1.1" 200 6118
INFO 2025-04-21 18:11:24,543 basehttp "GET /checkout/ HTTP/1.1" 200 9978
INFO 2025-04-21 18:11:32,655 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-21 18:14:51,824 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-21 18:14:52,319 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 18:14:55,840 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 18:14:57,390 basehttp "GET /cart/ HTTP/1.1" 200 6118
INFO 2025-04-21 18:14:58,101 basehttp "GET /checkout/ HTTP/1.1" 200 9978
INFO 2025-04-21 18:15:02,730 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-21 18:27:42,129 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-21 18:27:42,627 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 18:27:50,793 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 18:27:52,847 basehttp "GET /cart/ HTTP/1.1" 200 6118
INFO 2025-04-21 18:27:53,638 basehttp "GET /checkout/ HTTP/1.1" 200 9978
INFO 2025-04-21 18:28:00,417 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-21 18:35:18,577 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\models.py changed, reloading.
INFO 2025-04-21 18:35:19,075 autoreload Watching for file changes with StatReloader
ERROR 2025-04-21 18:35:22,611 log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: store_product.motor_rounds

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py", line 39, in store
    logger.info(f"Store page rendered with {len(products)} products")
                                            ~~~^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: store_product.motor_rounds
ERROR 2025-04-21 18:35:22,618 basehttp "GET / HTTP/1.1" 500 132222
INFO 2025-04-21 18:36:04,864 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\models.py changed, reloading.
INFO 2025-04-21 18:36:05,369 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 18:36:06,642 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 18:36:54,498 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 18:48:26,140 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\models.py changed, reloading.
INFO 2025-04-21 18:48:26,667 autoreload Watching for file changes with StatReloader
ERROR 2025-04-21 18:48:28,033 log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: store_product.motor_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py", line 39, in store
    logger.info(f"Store page rendered with {len(products)} products")
                                            ~~~^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: store_product.motor_id
ERROR 2025-04-21 18:48:28,040 basehttp "GET / HTTP/1.1" 500 131508
INFO 2025-04-21 18:49:50,798 autoreload Watching for file changes with StatReloader
ERROR 2025-04-21 18:49:53,237 log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: store_product.motor_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py", line 39, in store
    logger.info(f"Store page rendered with {len(products)} products")
                                            ~~~^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: store_product.motor_id
ERROR 2025-04-21 18:49:53,244 basehttp "GET / HTTP/1.1" 500 131508
INFO 2025-04-21 18:50:28,860 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 18:50:30,308 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 18:50:32,925 basehttp "GET /cart/ HTTP/1.1" 200 6118
INFO 2025-04-21 18:50:33,757 basehttp "GET /checkout/ HTTP/1.1" 200 9978
INFO 2025-04-21 18:50:40,793 basehttp "POST /process_order/ HTTP/1.1" 200 276
INFO 2025-04-21 18:55:47,647 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\models.py changed, reloading.
INFO 2025-04-21 18:55:48,236 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 18:58:34,857 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 19:56:06,075 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\models.py changed, reloading.
INFO 2025-04-21 19:56:06,607 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 19:59:08,196 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-21 19:59:08,673 autoreload Watching for file changes with StatReloader
ERROR 2025-04-21 20:10:31,849 log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: store_product.motor_rounds

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py", line 39, in store
    logger.info(f"Store page rendered with {len(products)} products")
                                            ~~~^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: store_product.motor_rounds
ERROR 2025-04-21 20:10:31,860 basehttp "GET / HTTP/1.1" 500 132150
INFO 2025-04-21 20:12:00,923 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 20:12:02,293 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 20:36:36,947 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 20:36:38,472 basehttp "GET / HTTP/1.1" 200 5174
INFO 2025-04-21 20:36:43,156 basehttp "GET /admin/ HTTP/1.1" 302 0
INFO 2025-04-21 20:36:43,182 basehttp "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4288
INFO 2025-04-21 20:36:46,903 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2025-04-21 20:36:46,921 basehttp "GET /admin/ HTTP/1.1" 200 13221
INFO 2025-04-21 20:36:47,044 basehttp "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2025-04-21 20:36:47,045 basehttp "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2025-04-21 20:36:47,045 basehttp "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2025-04-21 20:36:47,046 basehttp "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,029 basehttp "GET /admin/store/product/ HTTP/1.1" 200 11868
INFO 2025-04-21 20:36:52,061 basehttp "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,088 basehttp "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,089 basehttp "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,089 basehttp "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,090 basehttp "GET /static/admin/js/core.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,091 basehttp "GET /static/admin/js/actions.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,092 basehttp "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,093 basehttp "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,095 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:36:52,110 basehttp "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,120 basehttp "GET /static/admin/js/filters.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:52,146 basehttp "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
INFO 2025-04-21 20:36:56,915 basehttp "GET /admin/store/product/6/change/ HTTP/1.1" 200 14619
INFO 2025-04-21 20:36:56,939 basehttp "GET /static/admin/css/forms.css HTTP/1.1" 304 0
INFO 2025-04-21 20:36:56,939 basehttp "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:56,942 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:36:56,998 basehttp "GET /static/admin/js/change_form.js HTTP/1.1" 304 0
INFO 2025-04-21 20:36:56,999 basehttp "GET /static/admin/css/widgets.css HTTP/1.1" 304 0
INFO 2025-04-21 20:37:03,208 basehttp "GET /admin/store/product/ HTTP/1.1" 200 11868
INFO 2025-04-21 20:37:04,411 basehttp "GET /admin/store/product/4/change/ HTTP/1.1" 200 14644
INFO 2025-04-21 20:37:04,436 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:37:06,332 basehttp "GET /admin/store/product/ HTTP/1.1" 200 11868
INFO 2025-04-21 20:37:07,439 basehttp "GET /admin/store/product/3/change/ HTTP/1.1" 200 14614
INFO 2025-04-21 20:37:07,465 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:50:55,851 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\admin.py changed, reloading.
INFO 2025-04-21 20:50:56,355 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 20:50:58,896 basehttp "GET / HTTP/1.1" 200 5164
INFO 2025-04-21 20:51:07,146 basehttp "GET / HTTP/1.1" 200 5164
INFO 2025-04-21 20:51:10,412 basehttp "GET /admin/ HTTP/1.1" 200 13835
INFO 2025-04-21 20:51:13,136 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13355
INFO 2025-04-21 20:51:13,194 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:51:13,272 basehttp "GET /static/admin/img/search.svg HTTP/1.1" 200 458
INFO 2025-04-21 20:51:15,380 basehttp "GET /admin/store/product/6/change/ HTTP/1.1" 200 16264
INFO 2025-04-21 20:51:15,407 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:51:15,466 basehttp "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
INFO 2025-04-21 20:51:17,674 basehttp "GET /admin/store/motor/add/?_to_field=id&_popup=1 HTTP/1.1" 200 5058
INFO 2025-04-21 20:51:17,699 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:51:27,422 basehttp "POST /admin/store/motor/add/?_to_field=id&_popup=1 HTTP/1.1" 200 339
INFO 2025-04-21 20:51:27,434 basehttp "GET /static/admin/js/popup_response.js HTTP/1.1" 200 532
INFO 2025-04-21 20:51:30,767 basehttp "GET /admin/store/motor/add/?_to_field=id&_popup=1 HTTP/1.1" 200 5058
INFO 2025-04-21 20:51:30,791 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:51:33,915 basehttp "POST /admin/store/motor/add/?_to_field=id&_popup=1 HTTP/1.1" 200 339
INFO 2025-04-21 20:51:36,272 basehttp "GET /admin/store/motor/add/?_to_field=id&_popup=1 HTTP/1.1" 200 5058
INFO 2025-04-21 20:51:36,297 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:51:38,388 basehttp "POST /admin/store/motor/add/?_to_field=id&_popup=1 HTTP/1.1" 200 339
INFO 2025-04-21 20:52:05,987 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13355
INFO 2025-04-21 20:52:06,015 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:07,419 basehttp "GET /admin/store/product/4/change/ HTTP/1.1" 200 16385
INFO 2025-04-21 20:52:07,444 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:11,605 basehttp "GET /admin/store/order/ HTTP/1.1" 200 37551
INFO 2025-04-21 20:52:11,634 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:12,562 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13355
INFO 2025-04-21 20:52:12,592 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:15,649 basehttp "GET /admin/store/product/6/change/ HTTP/1.1" 200 16360
INFO 2025-04-21 20:52:15,676 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:17,843 basehttp "POST /admin/store/product/6/change/ HTTP/1.1" 302 0
INFO 2025-04-21 20:52:17,857 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13556
INFO 2025-04-21 20:52:17,948 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:18,000 basehttp "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
INFO 2025-04-21 20:52:19,915 basehttp "GET /admin/store/product/4/change/ HTTP/1.1" 200 16385
INFO 2025-04-21 20:52:19,943 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:22,153 basehttp "POST /admin/store/product/4/change/ HTTP/1.1" 302 0
INFO 2025-04-21 20:52:22,170 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13560
INFO 2025-04-21 20:52:22,279 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:24,043 basehttp "GET /admin/store/product/3/change/ HTTP/1.1" 200 16355
INFO 2025-04-21 20:52:24,069 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:52:27,441 basehttp "POST /admin/store/product/3/change/ HTTP/1.1" 302 0
INFO 2025-04-21 20:52:27,457 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13555
INFO 2025-04-21 20:52:27,567 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:57:10,392 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13355
INFO 2025-04-21 20:57:10,418 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:57:12,036 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13355
INFO 2025-04-21 20:57:12,058 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 20:57:13,264 basehttp "GET /admin/ HTTP/1.1" 200 13793
INFO 2025-04-21 20:57:15,015 basehttp "GET /admin/ HTTP/1.1" 200 13793
INFO 2025-04-21 20:57:17,262 basehttp "GET / HTTP/1.1" 200 5164
INFO 2025-04-21 20:57:19,753 basehttp "POST /update_item/ HTTP/1.1" 200 18
INFO 2025-04-21 20:57:19,771 basehttp "GET / HTTP/1.1" 200 5164
INFO 2025-04-21 20:57:20,183 basehttp "POST /update_item/ HTTP/1.1" 200 18
INFO 2025-04-21 20:57:20,203 basehttp "GET / HTTP/1.1" 200 5164
INFO 2025-04-21 20:57:21,081 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-21 20:57:22,098 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-21 20:57:30,476 basehttp "POST /process_order/ HTTP/1.1" 200 275
INFO 2025-04-21 20:59:32,721 basehttp "GET /admin/ HTTP/1.1" 200 13793
INFO 2025-04-21 20:59:34,712 basehttp "GET /admin/store/product/ HTTP/1.1" 200 13355
INFO 2025-04-21 20:59:34,747 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 11566
INFO 2025-04-21 21:09:25,096 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\myweb\settings.py changed, reloading.
INFO 2025-04-21 21:10:21,852 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 21:10:29,846 basehttp "GET / HTTP/1.1" 200 5164
INFO 2025-04-21 21:12:56,891 basehttp "GET / HTTP/1.1" 200 5164
INFO 2025-04-21 21:12:58,619 basehttp "GET /cart/ HTTP/1.1" 200 4873
INFO 2025-04-21 21:12:59,532 basehttp "GET /checkout/ HTTP/1.1" 200 9311
INFO 2025-04-21 21:13:05,703 basehttp "POST /process_order/ HTTP/1.1" 200 275
INFO 2025-04-21 21:17:10,979 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\models.py changed, reloading.
INFO 2025-04-21 21:17:11,513 autoreload Watching for file changes with StatReloader
ERROR 2025-04-21 21:17:13,306 log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: store_orderitem.motor_rounds

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py", line 36, in store
    data = cartData(request)
  File "D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\utils.py", line 58, in cartData
    cartItems = order.get_cart_items
                ^^^^^^^^^^^^^^^^^^^^
  File "D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\models.py", line 70, in get_cart_items
    total = sum([item.quantity for item in orderitems])
                                           ^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 400, in __iter__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\USER\AppData\Local\Programs\Python\Python313\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: store_orderitem.motor_rounds
ERROR 2025-04-21 21:17:13,314 basehttp "GET / HTTP/1.1" 500 140818
INFO 2025-04-21 21:17:50,412 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 21:17:51,907 basehttp "GET / HTTP/1.1" 200 5164
INFO 2025-04-21 21:43:38,498 autoreload D:\Desktop\PROJECTEND ไม่ไปหน้าชำระเงินสำเร็จ - Copy (2) - Copy - Copy - Copy\myweb\store\views.py changed, reloading.
INFO 2025-04-21 21:43:38,998 autoreload Watching for file changes with StatReloader
